features/gebruik_in_lopende_tekst.feature

```diff
@@ -446,68 +413,60 @@ Rule: een vrouw met een (ex)partner met een adellijke titel krijgt een hoffelijk
-    | naam                        | waarde    |
-    | burgerservicenummer         | 999992934 |
-    | geslachtsaanduiding (04.10) | V         |
-    En de persoon heeft de volgende 'naam' gegevens
-    | naam                                 | waarde        |
-    | voornamen (02.10)                    | Jo            |
-    | adellijke titel of predicaat (02.20) | BS            |
-    | voorvoegsel (02.30)                  | de            |
-    | geslachtsnaam (02.40)                | Boer          |
-    | aanduiding naamgebruik (61.10)       | <naamgebruik> |
-    En de persoon heeft een 'partner' met de volgende gegevens
-    | naam                        | waarde |
-    | geslachtsaanduiding (04.10) | M      |
-    En de 'partner' heeft de volgende 'naam' gegevens
-    | naam                                 | waarde  |
-    | adellijke titel of predicaat (02.20) | G       |
-    | voorvoegsel (02.30)                  | van den |
-    | geslachtsnaam (02.40)                | Aedel   |
-    Als personen wordt gezocht met de volgende parameters
-    | naam                | waarde                            |
-    | type                | RaadpleegMetBurgerservicenummer   |
-    | burgerservicenummer | 999992934                         |
-    | fields              | adressering.gebruikInLopendeTekst |
-    Dan heeft de response een persoon met de volgende 'adressering' gegevens
-    | naam                  | waarde                  |
-    | gebruikInLopendeTekst | <gebruikInLopendeTekst> |
-
-    Voorbeelden:
+    Gegeven de persoon met burgerservicenummer '000000279' heeft de volgende gegevens
+      | naam                                 | waarde        |
+      | geslachtsaanduiding (04.10)          | V             |
+      | voornamen (02.10)                    | Jo            |
+      | adellijke titel of predicaat (02.20) | BS            |
+      | voorvoegsel (02.30)                  | de            |
+      | geslachtsnaam (02.40)                | Boer          |
+      | aanduiding naamgebruik (61.10)       | <naamgebruik> |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                 | waarde  |
+      | geslachtsaanduiding (04.10)          | M       |
+      | adellijke titel of predicaat (02.20) | G       |
+      | voorvoegsel (02.30)                  | van den |
+      | geslachtsnaam (02.40)                | Aedel   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                            |
+      | type                | RaadpleegMetBurgerservicenummer   |
+      | burgerservicenummer | 000000279                         |
+      | fields              | adressering.gebruikInLopendeTekst |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                  | waarde                  |
+      | gebruikInLopendeTekst | <gebruikInLopendeTekst> |
+
+      Voorbeelden:
```

Comment:
Heeft het geen toegevoegde waarde om in bovenstaand scenario het geslacht variabel te maken ('M' en 'V').
Onderstaande tabel zou dan volgens mij als volgt worden:
```
      | geslacht | naamgebruik | gebruikInLopendeTekst                |
      | V        | E           | barones de Boer                      |
      | V        | P           | gravin van den Aedel                 |
      | V        | V           | gravin van den Aedel-barones de Boer |
      | V        | N           | barones de Boer-gravin van den Aedel |
      | M        | E           | baron de Boer                        |
      | M        | P           | de heer van den Aedel                |
      | M        | V           | de heer van den Aedel-baron de Boer  |
      | M        | N           | baron de Boer-van den Aedel          |
```

Reply:
Hier eigenlijk dezelfde reactie als bij de vorige opmerking. In de scenariotitel wordt de nadruk gelegd op de aanduiding naamgebruik, en die invloed wordt gechecked met de testcases. 

Als je overal geslacht ook variabel opneemt controleer je zaken driedubbel en maak je de scenario's complexer. 
In mijn beleving heeft het geen toegevoegde waarde omdat deze functionaliteit ook al wordt gechecked in de scenario's op regel 251 t/m 259, 289 t/m 296 , 317 t/m 326, 407 t/m 413 .